### PR TITLE
Add spelling corrections for defaoult and variants.

### PR DESCRIPTION
**Pull request by cfi-gb**
See e.g.:
- https://github.com/search?q=defaoult&type=code
- https://grep.app/search?q=defaoult (Has less hits)

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12454,6 +12454,9 @@ defailt->default
 defalt->default
 defalts->defaults
 defalut->default
+defaoult->default
+defaoulted->defaulted
```
[the remark this answers is not in the document]
Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt:12458]: One `defaoulted` has been found via:

https://www.google.com/search?q=%22defaoulted%22&nfpr=1

in some Reddit comment. As there are hundreds of hits for the `defaoult` typo it is probably just a matter of time until some one makes the same typo for `defaoulting` or `defaoulted`.

[the remark this answers is not in the document]
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:12458]: Can you add `defaoulting` then?

[the remark this answers is not in the document]
Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt:12458]: Ah, missed that one completely (i thought i had included it already :-/) and just had added it :+1:

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @cfi-gb.

**Pull request merged**